frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.tsx

```diff
@@ -71,7 +73,11 @@ export const PipelineVisualizationTask: React.FC<PipelineVisualizationTaskProp>
         },
       ]}
     >
-      <TaskComponent namespace={props.namespace} status={props.task.status} />
+      <TaskComponent
```

Comment:
Destructure `props` in line no. 64 to `({namespace, task})`

```suggestion
      <TaskComponent
        name={task.name}
        namespace={namespace}
        status={task.status}
      />

Reply:
Please note this is a bugfix. This is not meant for refactoring. So, the initial component implementation is kept as-is with minimal changes to avoid making any additional changes that are not necessary. :) Should I destructure the props, it will require me to make additional changes in places I wouldn't otherwise. cc @karthikjeeyar

Reply:
I would say fixing bug is a good opportunity to also look at refactoring some of the code within the scope of the bug to move towards following best practices more.  Otherwise there will always be a feature or a bug which will keep postponing the required refactoring in the name of technical debt.